data/test_yaml_stories/rules_without_stories.yml

```diff
@@ -34,3 +34,12 @@ rules:
   steps:
   - intent: explain
   - action: utter_explain_some_slot
+
+- rule: start optional preferences
+  steps:
+  - action: utter_request_optional
+  - or:
+    - intent: deny
+    - intent: inform
+  - action: form_plane_search_preference
```

Comment:
let's have some more common names in this test

Reply:
what do you mean?

Reply:
it's weird to have these arbitrary concrete action names in a test